lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +142,23 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+          options[:color_mode] = :automatic
+        end
+
+        parser.on('--force-color', '--force-colour', 'Force the output to be in color, even if the output is not a TTY') do |_o|
+          if options[:color_mode] == :disabled
+            abort "Please only use one of --force-color and --no-color"
```

Comment:
Can you put backticks around `--force-color` and `--no-color`?  We generally do that in messages like this.